packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -114,13 +114,68 @@ function useGlobalStylesOpenCssCommands() {
 	};
 }
 
-export function useCommonCommands() {
+function useGlobalStylesOpenRevisionsCommands() {
 	const { openGeneralSidebar, setEditorCanvasContainerView, setCanvasMode } =
 		unlock( useDispatch( editSiteStore ) );
+	const { getCanvasMode } = unlock( useSelect( editSiteStore ) );
 	const { params } = useLocation();
 	const isMobileViewport = useViewportMatch( 'medium', '<' );
-	const isListPage = getIsListPage( params, isMobileViewport );
-	const isEditorPage = ! isListPage;
+	const isEditorPage = ! getIsListPage( params, isMobileViewport );
+	const history = useHistory();
+	const revisions = useSelect(
+		( select ) =>
+			select( coreStore ).getCurrentThemeGlobalStylesRevisions(),
+		[]
+	);
```

Comment:
Sorry, I've one final tiny nitpick 😅

As we don't need the object revisions, let's return the primitive `hasRevisions` boolean from the selector.